docs/helm/examples/with-aws-services.md

```diff
@@ -102,16 +102,67 @@ The following sections describe how to setup the AWS services and highlights the
 
 4. Finally, take a note of the OpenWire Endpoint displayed in the "Connections" section
 
+>  :warning: make sure to use a failover URL. If you chose to use a "Single -instance broker", still wrap its URL in `failover:()`.
+
 ## Deploy
 
 In order to use the S3 connector and external database options, the S3 connector AMP and database drivers are required, respectively. Fortunately, a Docker image has been pre-packaged with the artifacts and can be used as-is for our deployment. To use the image we will override the `repository.image.repository` property.
 
 To use the S3 connector, RDS and Amazon MQ we have to disable the internal default components via the Helm "set" command and additionally provide the service endpoints and credentials we made a note of in the previous sections.
 
-When we bring all this together we can deploy ACS using the command below (replacing all the `YOUR-XZY` properties with the values gathered during the setup of the services):
+When we bring all this together we can deploy ACS using the command below (replacing all the `YOUR-XZY` properties with the values gathered during the setup of the services).
+Edit your `values.yml` file so it contains below elements:
+
+```yaml
+externalPort: 443
+externalProtocol: https
+externalHost: acs.YOUR-DOMAIN-NAME
+persistence:
+  enabled: true
+  storageClass:
+    enabled: true
+    name: nfs-client
+global:
+  alfrescoRegistryPullSecrets: quay-registry-secret
+repository:
+  image:
+    repository: alfresco-content-repository-aws
+s3connector:
+  enabled: true
+  config:
+    bucketName: YOUR-BUCKET-NAME
+    bucketLocation: YOUR-AWS-REGION
+postgresql:
+  enabled: false
+database:
+  external: true
+  driver: org.postgresql.Driver
+  url: jdbc:postgresql://YOUR-DATABASE-ENDPOINT:5432/
+  user: alfresco
+  password: YOUR-DATABASE-PASSWORD
+messageBroker: &acs_messageBroker
+  url: YOUR-MQ-ENDPOINT
+  user: alfresco
+  password: YOUR-MQ-PASSWORD
+```
+
+Then you can deploy using:
+
+```bash
+helm -n alfresco install acs ./alfresco/alfresco-content-services --atomic --timeout 10m0s
+```
+
+If you're deploying from the registry of charts you can't update the `values.yml` file. Instead you either:
+
+  * use a local copy of the `values.yml` file amended as shown above
+  * use `--set` options to pass individual values.
+
+Note however that the main `values.yml` file uses [YAML anchors and aliases](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) to propagate configuration to subcharts if needed.
+Using `--set` do not fill the yaml file with passed values and so aliases will never point to a node filled with the values passed by the command line.
+That means when using command line you must also pass aliased values as shown below (at the end of the command):
 
 ```bash
-helm install acs alfresco/alfresco-content-services \
+helm -n alfresco install acs alfresco-stable/alfresco-content-services --atomic --timeout 10m0s
```

Comment:
alfresco-stable?

Reply:
that used to be how the doc referenced the alfresco helm repo but looks like it's been changed to `alfresco` when we decided to give up on incubator repo.
Changing that.